src/vorta/i18n/ts/vorta.de.ts

```diff
@@ -449,121 +434,116 @@
     </message>
     <message>
         <location filename="../../assets/UI/archivetab.ui" line="97"/>
+        <source>Mounted</source>
+        <translation>Eingehängepunkt</translation>
```

Comment:
that seems strange at least.

Reply:
Haha. Should be "Einhängepunkt"

Reply:
It shows the mount path. So I figured it's better than just "Eingehängt", which implies yes/no.

Reply:
Fixed in TX.

Reply:
Well, if it is the wrong term, the source string should be changed or it might be strange in all other (more close) translations.

Reply:
True. Let's use "Mount Point". This changed only yesterday.

Reply:
also, the question is if that is just a description of the path "oh, this is a (potential) mount point" or a description of the current status "this is currently mounted".

Reply:
Changed